src/bat/inswin31.bat

```diff
@@ -1,4 +1,5 @@
 @echo off
 unix -s dosemu-preinstallwin31

```

Comment:
You need to check errorlevel
here and handle the failure.

Reply:
I guess that should especially also happen for the line below to make sure setup ran successfully?

Reply:
Yes, indeed.